src/Uno.UI/UI/Xaml/Style/Generic/Generic.xaml

```diff
@@ -2098,7 +2098,7 @@
 	<SolidColorBrush x:Key="IOSTimePickerDismissButtonForegroundBrush"
 					 Color="#055bb7" />
```

Comment:
should these ones be theme-specific as well?

Reply:
I feel like the native ios blue color (the color in question) works for both light/dark themes. 
However I could add them as is to the theme dictionaries if you think its pertinent

Reply:
If the color works in both modes, it's fine like this.